### PR TITLE
feat(docs): update event_topic of data_integrations

### PR DESCRIPTION
**Pull request by luoyq0629**

### CODE REVIEW

**Review by zibuyu1995 — APPROVED** [review]

File: en_US/rule_engine/rule_engine_events.md [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
+| $events/client_connected     | Client connect                  |
+| $events/client_disconnected  | Client disconnect               |
+| $events/message_delivered    | Message delivery                |
+| $events/message_acked        | Message acknowledged            |
+| $events/message_dropped      | Message dropped when routing    |
+| $events/delivery_dropped     | Message dropped when delivering |
+| $events/session_subscribed   | Subscribe                       |
+| $events/session_unsubscribed | Unsubscribe                     |
 
 This article will introduce the usage of event topics, and the meaning of each field from three aspects: client connect and disconnect events, message events, topic subscribe and unsubscribe events.
```
Comment (Meggielqk) [inline en_US/rule_engine/rule_engine_events.md:16]: This page introduces the usage of event topic and the meaning of each field of the following event types: 

- Client connect and disconnect events 
- Message events
- Topic subscribe and unsubscribe events

File: en_US/rule_engine/rule_engine_events.md [hunk 2]
```diff
@@ -180,25 +181,59 @@ The topic of client disconnect is `$events/message_dropped`, click on Rules on t
...
-| topic               | MQTT topic                                    |
-| username            | Current MQTT username                         |
+| Field               | Explanation                                                                                                                                                                                                                  |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id                  | MQTT message id                                                                                                                                                                                                              |
+| reason              | Reasons of dropping, possible reasons: <br>no_subscribers: no clients subscribes the topic<br>receive_maximum_exceeded: awaiting_rel queue is full<br>packet_identifier_inuse: send a qos2 message with unreleased packet ID |  |
```
Comment (Meggielqk) [inline en_US/rule_engine/rule_engine_events.md:191]: Reasons of dropping; possible reasons: <br />no_subscribers: no clients subscribes the topic<br />receive_maximum_exceeded: awaiting_rel queue is full<br />packet_identifier_inuse: send a qos2 message with unreleased packet ID

File: en_US/rule_engine/rule_engine_events.md [hunk 3]
```diff
@@ -180,25 +181,59 @@ The topic of client disconnect is `$events/message_dropped`, click on Rules on t
...
+| timestamp           | Event trigger time(millisecond)                                                                                                                                                                                              |
+| publish_received_at | Time when PUBLISH message reaches Broker (ms)                                                                                                                                                                                |
+| node                | Node name of the trigger event                                                                                                                                                                                               |
+
+### Delivery_dropped
+
+The topic of delivery_dropped is `$events/delivery_dropped, trigger rule when subscriber's message queue is full. Click on Rules on the left menu bar, click on Create, and enter the following rule to match the SQL statement:
```
Comment (Meggielqk) [inline en_US/rule_engine/rule_engine_events.md:206]: The topic of delivery_dropped is `$events/delivery_dropped`, which triggers the rule when subscriber's message queue is full. Click **Rules** from the left navigation menu. On the Rules page, click **Create**, and enter the following rule to match the SQL statement:

File: en_US/rule_engine/rule_engine_events.md [hunk 4]
```diff
@@ -180,25 +181,59 @@ The topic of client disconnect is `$events/message_dropped`, click on Rules on t
...
+
+   ```sql
+    SELECT
+        *
+    FROM
+        "$events/delivery_dropped"
+   ```  
+
+You can test the SQL, view the return field after firing this rule.
```
Comment (Meggielqk) [inline en_US/rule_engine/rule_engine_events.md:215]: You can test the SQL, and view the return field after firing this rule.

What does "firing" mean here?

File: zh_CN/rule_engine/rule_engine_events.md [hunk 5]
```diff
@@ -181,22 +182,55 @@ EMQX Cloud 数据集成 提供了以下七个可用的事件订阅主题。
...
-| publish_received_at | PUBLISH 消息到达 Broker 的时间 (ms) |
-| qos                 | MQTT 消息的 QoS                     |
-| reason              | 消息丢弃原因                        |
-| timestamp           | 事件触发时间 (ms)                   |
-| topic               | MQTT 主题                           |
-| username            | 用户名                              |
+| 字段                | 释义                                                                                                                                                     |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| clientid            | 客户端 ID                                                                                                                                                |
+| reason              | 消息丢弃原因，可能的原因：<br>no_subscribers: 没有订阅者<br>receive_maximum_exceeded: awaiting_rel 队列已满<br>packet_identifier_inuse: 消息 ID 已被使用 |
```
Comment (Meggielqk) [inline zh_CN/rule_engine/rule_engine_events.md:188]: 消息丢弃原因，可能的原因：<br />no_subscribers: 没有订阅者<br />receive_maximum_exceeded: awaiting_rel 队列已满<br />packet_identifier_inuse: 消息 ID 已被使用

File: zh_CN/rule_engine/rule_engine_events.md [hunk 6]
```diff
@@ -181,22 +182,55 @@ EMQX Cloud 数据集成 提供了以下七个可用的事件订阅主题。
...
+
+使用 SQL 测试，查看触发该规则后返回字段，也可在 SQL 语句中使用 SELECT 和 WHERE 子句进行筛选与过滤。
+![消息丢弃](./_assets/rule_engine_event_delivery_dropped.png)
+
+各字段释义如下。
 
+| 字段                | 解释                                                                                                                                                                                  |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id                  | MQTT 消息 ID                                                                                                                                                                          |
+| reason              | 消息丢弃原因，可能的原因：<br>queue_full: 消息队列已满(QoS>0)<br>no_local: 不允许客户端接收自己发布的消息<br>expired: 消息或者会话过期<br>qos0_msg: QoS0 的消息因为消息队列已满被丢弃 |
```
Comment (Meggielqk) [inline zh_CN/rule_engine/rule_engine_events.md:220]: 消息丢弃原因，可能的原因：<br />queue_full: 消息队列已满(QoS>0)<br />no_local: 不允许客户端接收自己发布的消息<br>expired: 消息或者会话过期<br />qos0_msg: QoS0 的消息因为消息队列已满被丢弃

**Pull request merged**